Vagrantfile

```diff
@@ -24,53 +24,72 @@ Vagrant.configure("2") do |config|
 # Get the VM up-to-date
     sudo apt-get update
 
-# Install opam and things for compiling xapi
-    sudo apt-get install -y opam m4 libxen-dev git
+# Install opam and dependencies for compiling xapi
+    sudo apt-get install -y opam m4 libxen-dev
     opam init -a --compiler=4.02.3 -y
+    eval `opam config env`
     opam remote add xs-opam git://github.com/xapi-project/xs-opam
```

Comment:
I think this step also requires `git`, so we have to install `git` before this line? Or did it work without git :o?